__tests__/buildTranslationFiles.spec.ts

```diff
@@ -54,36 +56,55 @@ type TranslationCategory =
 interface assertTranslationParams {
   type: TranslationCategory;
   expected: object;
+  outputFormat: 'json' | 'pot';
   path?: string;
 }
 
-function assertTranslation({ type, expected, path }: assertTranslationParams) {
-  expect(loadTranslationFile(type, path)).toEqual(expected);
+function assertTranslation({
+  type,
+  expected,
+  path,
+  outputFormat,
+}: assertTranslationParams) {
+  expect(loadTranslationFile(type, path, outputFormat)).toEqual(expected);
 }
 
 function assertPartialTranslation({
   type,
   expected,
   path,
+  outputFormat,
 }: assertTranslationParams) {
-  expect(loadTranslationFile(type, path)).toMatchObject(expected);
+  expect(loadTranslationFile(type, path, outputFormat)).toMatchObject(expected);
 }
 
-function loadTranslationFile(type: TranslationCategory, path?: string) {
-  return fs.readJsonSync(`./${sourceRoot}/${type}/i18n/${path || ''}en.json`);
+function loadTranslationFile(
+  type: TranslationCategory,
+  path: string,
+  outputFormat: 'json' | 'pot'
+) {
+  return getCurrentTranslation(
+    `./${sourceRoot}/${type}/i18n/${path || ''}en.${outputFormat}`,
+    outputFormat
+  );
 }
 
 function removeI18nFolder(type: TranslationCategory) {
   fs.removeSync(`./${sourceRoot}/${type}/i18n`);
 }
 
-describe('buildTranslationFiles', () => {
+const formats: ['json' | 'pot'][] = [['json'], ['pot']];
```

Comment:
Why is this an array of arrays? 🤔

Reply:
Leftover, I will change it to a simple array.